client/src/App.tsx

```diff
@@ -102,6 +103,9 @@ export default function App() {
 
   const removeFromQueue = async (user: QueueEntry) => {
     console.log(timestamp(), 'Removing ' + user.username + ' from queue')
+    if(currentHolder?.id === getPrivateKey()){
```

Comment:
```suggestion
    if (currentHolder?.id === getPrivateKey()) {
```
Last ned prettier xD